src/Integration/Binding/RuleSetReferenceChecker.cs

```diff
@@ -23,12 +23,21 @@
 using System.IO;
 using System.Linq;
 using EnvDTE;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
 using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
-using SonarLint.VisualStudio.Integration.Helpers;
 using SonarLint.VisualStudio.Integration.Resources;
 
 namespace SonarLint.VisualStudio.Integration.Binding
 {
+    // TODO - CM cleanup - this class might be useful when cleaning up "old" Connected Mode settings.
```

Comment:
FYI I've added card to the board to remind us to check that all of the `// TODO - CM cleanup` markers have been handled before ending the sprint.